waitsensor/waitsensor.go

```diff
@@ -172,6 +171,14 @@ func (bb BoundingBox) Crop(img image.Image) image.Image {
 	return croppedImg
 }
 
+type syncValues struct {
+	mu              sync.RWMutex
```

Comment:
I've never understood our heavy use of RWMutex. They let you read the value from multiple threads simultaneously, but in practice no one does that in these use cases, and locking one of these is a much heavier operation than locking a normal Mutex. Why bother with the RWMutex at all? My instinct is to just use a `sync.Mutex` unless there's a compelling reason to do something else (e.g., you're building a database).

I dunno; leave as-is if you want. :shrug: